hub/static/js/all.js

```diff
@@ -153,18 +155,22 @@ var setUpMap = function() {
         attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
     }).addTo(map)
 
-    window.geojson = L.geoJson(constituencies, {
-        style: getFeatureStyle,
-        onEachFeature: function(feature, layer){
-            layer.on({
-                mouseover: highlightFeature,
-                mouseout: unhighlightFeature,
-                click: function(){
-                    window.location.href = '/area'
-                }
-            })
-        }
-    }).addTo(map)
+    $.getJSON("/filter_areas/", function(data) {
+        console.log(data);
```

Comment:
Can see why this was left in, but we should probably remove it before we forget!